eng/templates/Azure.Management.Template/src/Azure.Management.Template.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>0.1.0</Version>
+    <PackageId>Azure.Management.Template</PackageId>
+    <Description>Azure management client SDK for AzureRP </Description>
+    <PackageTags>azure;management;ProviderNameLowercase</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>($NoWarn);CA1812;</NoWarn>
```

Comment:
If we need to disable this warning for everyone we should probably also move it into the common props.

Reply:
CA1812 and AZC0012 specific affect mgmt plane right now. I have moved into common props.